utils/http.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/companieshouse/chs.go/log"
+)
+
+// ResponseResource is the object returned in an error case
+type ResponseResource struct {
+	Message string `json:"message"`
+}
+
+// NewMessageResponse - convenience function for creating a response resource
+func NewMessageResponse(message string) *ResponseResource {
+	return &ResponseResource{Message: message}
+}
+
+// WriteJSONWithStatus writes the interface as a json string with the supplied status.
+func WriteJSONWithStatus(w http.ResponseWriter, r *http.Request, data interface{}, status int) {
```

Comment:
It looks like this function is now being used in a few services. Would it be best to now move it into chs.go?